### PR TITLE
doc: add rbd new trash cli and cleanups  in release-notes.rst

### PR DESCRIPTION
**Pull request by Songweibin**
[http://tracker.ceph.com/issues/20702](url)
Add rbd trash commands.

Signed-off-by: songweibin <song.weibin@zte.com.cn>

### CODE REVIEW

File: doc/release-notes.rst [hunk 1]
```diff
@@ -129,7 +129,7 @@ Major Changes from Kraken
   * Improved discard handling when the object map feature is enabled.
   * rbd CLI ``import`` and ``copy`` commands now detect sparse and
     preserve sparse regions.
-  * Images and Snapshots will now include a creation timestamp
+  * Images and Snapshots will now include a creation timestamp.
```
Comment (tchaikov) [inline doc/release-notes.rst]: could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

Comment (Songweibin) [inline doc/release-notes.rst]: done, thanks.

**Comment by scienceluo** [comment]
According to the organizationmap , Please update your Signed-off-by info to:

Signed-off-by: Song Weibin <song.weibin@zte.com.cn>

**Comment by dillaman** [comment]
@Songweibin can you add the ```doc: ``` prefix to all your commit subjects?

**Comment by Songweibin** [comment]
@dillaman done, thanks.

**Pull request merged**